### PR TITLE
Minor Typo in zh-cn docs fixed

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #36060

### CODE REVIEW

**Comment by taniaduggal** [comment]
lgtm


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: content/zh-cn/docs/reference/kubernetes-api/common-parameters/common-parameters.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ fieldManager 是与进行这些更改的参与者或实体相关联的名称。
 A selector to restrict the list of returned objects by their fields. Defaults to everything.
 <hr>
 -->
-根据返回对象的字段限制返回对象列表的选择器。默认为返回所有字段。
+根据返回对象的字段限制返回对象列表的选择器。默认为返回所有对象。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/common-parameters/common-parameters.md:86]:
```suggestion
这个选择算符根据对象字段限制返回的对象列表。默认为返回所有对象。
```
The first clause looks like a bit tongue twister, but I have no good ideas yet, only a suggestion for your reference.

Comment (NitishKumar06) [reply content/zh-cn/docs/reference/kubernetes-api/common-parameters/common-parameters.md:86]: @seokho-son WDYT of this?

Comment (Sea-n) [reply content/zh-cn/docs/reference/kubernetes-api/common-parameters/common-parameters.md:86]: Not sure why @NitishKumar06 you mention @seokho-son here.

He is a Korean speaker, not a Chinese speaker.

Comment (seokho-son) [reply content/zh-cn/docs/reference/kubernetes-api/common-parameters/common-parameters.md:86]: I am not eligible for reviewing this PR. :)

Comment (NitishKumar06) [reply content/zh-cn/docs/reference/kubernetes-api/common-parameters/common-parameters.md:86]: @Sea-n My BAD! I got confused.

Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/common-parameters/common-parameters.md:86]:
```suggestion
限制所返回对象的字段的选择器。默认为返回所有字段。
```

Comment (NitishKumar06) [reply content/zh-cn/docs/reference/kubernetes-api/common-parameters/common-parameters.md]: DONE!

**Pull request merged**